nixos/modules/services/x11/desktop-managers/default.nix

```diff
@@ -26,6 +26,31 @@ in
 
     services.xserver.desktopManager = {
 
+      wallpaper = {
+        mode = mkOption {
+          type = types.enum [ "center" "fill" "max" "scale" "tile" ];
+          default = "scale";
+          example = "fill";
+          description = ''
+            Wallpaper image mode:
```

Comment:
I do not think there is any documentation about the fact that `~/.background-image` is used for the wallpaper image.
I think it would be good to make a follow-up patch to add this information as part of this description. What do you think?